style.css

```diff
@@ -0,0 +1,71 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+body{
+    min-height: 100vh;
+    background: #d3d3d3;
+    font-family: 'Roboto' , sans-serif;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+::-webkit-scrollbar{
+    display: none;
+}
+.paper{
+    position: relative;
```

Comment:
group similar properties as @syedabdullahali has mentioned

Reply:
i just arrange properties in a group...
little bit confuse in grouping if their is any specific format of it please explain , thank you

Reply:
you can go through senior members projects and look how they have grouped. You can check mine also.

Reply:
okay ill go through seniors projects for better understanding of mine.